lib/active_storage_validations/aspect_ratio_validator.rb

```diff
@@ -4,6 +4,8 @@
 
 module ActiveStorageValidations
   class AspectRatioValidator < ActiveModel::EachValidator # :nodoc
+    include OptionProcUnfolding

```

Comment:
Can we reuse the code and logic from active model? Since we depend on it anyway, why not use that a bit more. For example the https://github.com/rails/rails/blob/d695972025146713fae9a089dcaf2239ede354d4/activemodel/lib/active_model/validations/resolve_value.rb#L6 method might be useful.

Reply:
That method unfortunately isn't yet part of any rails stable version. We're downwards compatible to 5.2 at the moment, right? We might copy it though...